src/App.js

```diff
@@ -9,28 +9,54 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData,
-      view: 'mainPage'
+      // movieData: movieData,
+      movieData: {
```

Comment:
I'm curious about this nested bit of state - what's the advantage of this over just storing an array of movies in state directly?